content/calc/limit-props.txt

```diff
@@ -91,7 +118,7 @@ and the smaller interval is a subset of the bigger interval;
 this means that the bigger interval contains all numbers of the smaller interval.
```

Comment:
This is actually used in the boundedness page as well. Should probably move the explanation there and point to it from here